src/components/AddCompanies/AddCompanies.jsx

```diff
@@ -0,0 +1,93 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Button, Form, FormGroup, ControlLabel, FormControl, Col } from 'react-bootstrap';
+import './styles.less';
+
+const propTypes = {
+    addCompany: PropTypes.func.isRequired,
+};
+
+class AddCompanies extends Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			name: '',
+			address: '',
+			phone: '',
+			revenue: '',
+		};
+
+		this.addNewCompany = this.addNewCompany.bind(this);
+		this.handleFormControlChange = this.handleFormControlChange.bind(this);
+	}
+
+	addNewCompany(e) {
+		e.preventDefault();
+		const newCompany = {...this.state};
+		this.props.addCompany(newCompany)
+	}
+
```

Comment:
This is my `in Component` method that will use the addCompany method passed down from the container. I'm using it here because I wanted to ensure I prevented any submission (Bootstrap) and also in the event I need to massage the data I can do that before using `this.props.addCompany`.